sci-misc/boinc/boinc-7.16.14.ebuild

```
@@ -107,16 +107,17 @@ src_prepare() {
 
 	eautoreconf
 
-	use X && setup-wxwidgets
-
 	# bug #732024
 	if test "x$(get_libdir)" = "xlib64"; then
 	    sed -i -e 's,/lib\([ /;:"]\),/lib64\1,g' configure || die
 	fi
 }
 
 src_configure() {
-	LDFLAGS="-L${EPREFIX}/usr/$(get_libdir) -L${EPREFIX}/$(get_libdir) ${LDFLAGS}" \
+	use X && setup-wxwidgets
+
+	append-libs "-L${EPREFIX}/usr/$(get_libdir)" "-L${EPREFIX}/$(get_libdir)"
```

@Yamakuzure thanks. Last ultra tiny niggle: Because you're just linking to from certain target directories, we should be using `${ESYSROOT}` here, that makes cross-compilation more likely to work.
```suggestion
	append-libs "-L${ESYSROOT}/usr/$(get_libdir)" "-L${ESYSROOT}/$(get_libdir)"
```
can you do that for all ebuilds (seeing they're all EAPI 7 now)?

---

I just wanted to answer that I have noted this down for the next bump, but I saw that you already changed that. Thank you very much!